chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertifiedProductController.java

```diff
@@ -96,9 +92,15 @@
 @RestController
 @RequestMapping("/certified_products")
 public class CertifiedProductController {
-    private static final Logger LOGGER = LogManager.getLogger(CertifiedProductController.class);
+    private static Logger LOGGER = LogManager.getLogger(CertifiedProductController.class);
```

Comment:
I've been replacing these lines with an `@Log4j2` Lombok annotation. Not required